create-module.sh

```diff
@@ -262,25 +308,41 @@ EOF
 # Create README.md
 readme="$cssPath/README.md"
 echo -e "Creating ${CYAN}$readme${NC}"
+
+if [ $stable ] ; then
 cat > $readme << EOF
 # Canvas Kit CSS $upperName
 EOF
+else
```

Comment:
Similar here, it's almost like the title can be extracted out. Either way, these are all nits, no big deal